### DISCUSSION TITLE
Toggle 컴포넌트는 input이다 vs button이다

### DISCUSSION THREAD

**Discussion by eugene028** [gdg-hongik-univ/wow-design-system#26] [General]
Toggle 컴포넌트는 `input`으로 정의되어야 할까요... `Button`으로 정의되어야 할까요? 그것이 문제로다..!

## 관련 이슈 
#22 

## 관련 래퍼런스
🔗 [MUI- Toggle Button](https://mui.com/material-ui/react-toggle-button/)
🔗 [MUI- Switch](https://mui.com/material-ui/react-switch/)
🔗 [Stackoverflow- <input type='button'>과 <button>은 무슨 차이가 있나요?](https://stackoverflow.com/questions/469059/button-vs-input-type-button-which-to-use)

## 논의 기록 
2024년 05월 26일 일요일 회의에서 더욱 다루어 볼 것 .. with 디자이너 예영님

**Comment by developomp** [comment]
`input`으로 정의되었습니다!
